src/components/TeamSection/TeamInformation.ts

```diff
@@ -1,13 +1,20 @@
 import member_0 from './Images/Joel_Happ.jpeg'
-import member_1 from './Images/Valeriya_Volkova.png'
-import member_2 from './Images/Arpith_Indukuri.jpg'
+import member_1 from './Images/Naureen_Othi.jpg'
+import member_2 from './Images/Fedor_Prokopchuk.jpg'
+//import member_1 from './Images/Valeriya_Volkova.png'
+//import member_2 from './Images/Arpith_Indukuri.jpg'
 import member_3 from './Images/Luke_Hollinda.jpg'
 import member_4 from './Images/Amanda_Nguyen.jpg'
 import member_5 from './Images/Tyler_Chan.jpg'
 import member_6 from './Images/Sareen_Singh.jpg'
-import member_7 from './Images/Armeen_Rashidian.jpg'
-import member_8 from './Images/Felix_Vaughan.jpg'
-import member_9 from './Images/Darryl_Huët.jpg'
+import member_7 from './Images/Nurgul_Akhshatayeva.jpg'
```

Comment:
You don't need to change this - but I wanted to note:
Importing everyone's image as member_X could make it confusing to know which image is associated with who. This isn't a huge deal, but it if we imported them something like 
``` import nurgul_jpg from './Images/Nurgul_Akhshatayeva.jpg``` it would be more maintainable for when we make changes in the future